Source/Widgets/Cesium3DTilesInspector/Cesium3DTilesInspectorViewModel.js

```diff
@@ -998,7 +998,7 @@ define([
         var loadSiblings = knockout.observable();
         knockout.defineProperty(this, 'loadSiblings', {
             get : function() {
-                loadSiblings();
+                return loadSiblings();
```

Comment:
So was this just an obvious bug we missed?  This getter always returned undefined, which meant it was always false.  Who is most familiar with this code? CC @lilleyse @ggetz @hpinkos

Reply:
Looks like it was a bug that we missed.